4-projects/README.md

```diff
@@ -75,8 +75,7 @@ This pipeline can be utilized for deploying resources in projects across develop
    gcloud access-context-manager perimeters list --policy ACCESS_CONTEXT_MANAGER_POLICY_ID --format="value(name)"
    ```
 
-**Troubleshooting:**
-If you do not have access to run the commands above and you are in the organization admins group, you can append `--impersonate-service-account=org-terraform@<SEED_PROJECT_ID>.iam.gserviceaccount.com` to run the command as the Terraform service account.
+**Note:** Please refer to [troubleshooting](../docs/TROUBLESHOOTING.md) if you run into issues during this step.
```

Comment:
When I read the rendered markdown, it seems easy to miss. Can we put a h3 heading under prereqs called troubleshooting and then the text `Please refer to [troubleshooting](../docs/TROUBLESHOOTING.md) if you run into issues during this step.`

Reply:
done